src/frontend/src/pages/GanttPage/GanttChartPage.tsx

```diff
@@ -265,6 +322,15 @@ const GanttChartPage: FC = () => {
           showWorkPackagesMap={showWorkPackagesMap}
           setShowWorkPackagesMap={setShowWorkPackagesMap}
         />
+        {Array.from(groupedEventChanges.entries()).map(([changeId, change]) => {
+          return (
```

Comment:
Verify here as well that the activeids you're using to check the changeId is the correct list